torch/jit/__init__.py

```diff
@@ -1651,6 +1682,10 @@ def _convert_to_script_module(mod, methods=None):
     `('forward',)`. Methods accessed in forward are scripted on demand if
     `_enable_recursive_script()` is used.
     """
+    if isinstance(mod, (ModuleList, Sequential)):
```

Comment:
Why isn't this change not guarded by a `torch._C._jit_recursive_script` check

Reply:
There's only 1 guard with that check in `script()` before it calls `_convert_to_script_module`, adding it everywhere would be cumbersome to delete later